app/src/main/res/values/styles.xml

```diff
@@ -93,4 +93,8 @@
         <item name="android:textStyle">normal</item>
         <item name="android:textColor">#FFFFFF</item>
     </style>
+
+    <style name="SwitchStyle" parent="AppTheme">
```

Comment:
Should we inherit from `Widget.AppCompat.CompoundButton.Switch`?

Reply:
No, this is to override a theme, where the colorAccent is defined, without updating the whole app's theme. I originally tried setting the thumb/track through a selector, but 1) this way is much simpler, and 2) that way doesn't work for r22 (and only works on 23+).